smart-contracts/scripts/auction_marketplace_deployment/3_create_collections_on_marketplace.js

```diff
@@ -73,14 +73,14 @@ async function main() {
   const mainnet_startTime = '1610740800'; // Jan 15, 8pm
 
   // Use the single auction id processed in the last script to build auction id specific collections in this script
-  const auctionId_3dBehemoth = 0; // transformation TODO update from result of last script (important)
-  const auctionId_yekaterina = 0; // harajuku TODO update from result of last script (important)
-  const auctionId_lorena = 0; // few charm TODO update from result of last script (important)
-  const auctionId_nina = 0; // cosmic one TODO update from result of last script (important)
+  const auctionId_3dBehemoth = 39; // transformation TODO update from result of last script (important)
+  const auctionId_yekaterina = 40; // harajuku TODO update from result of last script (important)
+  const auctionId_lorena = 41; // few charm TODO update from result of last script (important)
+  const auctionId_nina = 42; // cosmic one TODO update from result of last script (important)
 
   // Next step is mint collections and open buy offers, run 1 at a time in production in case something drops
   const collectionUris = [
-    {
+    /*{
```

Comment:
Remove these comments, leave it as is so we have traceability of what we did before. We probably won't use the script again though